tests/cleo_tests/cleo_tester.inc

```diff
@@ -101,59 +101,173 @@ hex
     00(256)
 end
 
-:_cleo_tester_fail
-int test_index = _cleo_tester_read_var(VAR_TEST_INDEX)
-int test_name = get_label_pointer @_cleo_tester_spec_name
-int assert_index = _cleo_tester_read_var(VAR_ASSERT_INDEX)
-breakpoint "~r~~h~~h~~h~Test #%d Assert #%d FAILED! %d Expected, %d Actual" test_index assert_index {val1} 0@ {val2} 1@
-terminate_this_custom_script
+function _cleo_tester_fail
+    int test_index = _cleo_tester_read_var(VAR_TEST_INDEX)
+    int test_name = get_label_pointer @_cleo_tester_spec_name
+    int assert_index = _cleo_tester_read_var(VAR_ASSERT_INDEX)
+    trace "~r~~h~~h~~h~Test #%d Assert #%d FAILED!" test_index assert_index
+end
 
 function _cleo_tester_increment_assert
     int index = _cleo_tester_read_var(VAR_ASSERT_INDEX)
     index++
     _cleo_tester_write_var(VAR_ASSERT_INDEX, index)
 end
 
+/// checks if bool value is true (different than 0)
+function assert_true(flag: int)
+    _cleo_tester_increment_assert
+	if
+		flag == false
+	then
+        _cleo_tester_fail
+        trace "TRUE expected~n~%d occured" flag
+        breakpoint
```

Comment:
maybe ?

```
breakpoint "TRUE expected~n~%d occured" flag
```

Reply:
breakpoint causes to put the message in the middle of log entry

Reply:
sorry, what does it mean?

Reply:
Breakpoint prints multiple lines on the screen, and if I remember correctly it incorporates user's message somewhere inside the longer text ("breakpoint named X was hit"). It also changes color of the message.